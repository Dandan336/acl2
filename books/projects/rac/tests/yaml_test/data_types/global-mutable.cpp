// RAC begin

int a = 2;

int foo()
{
  return a;
}

// RAC end
