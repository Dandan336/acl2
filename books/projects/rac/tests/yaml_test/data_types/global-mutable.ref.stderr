global-mutable.cpp:3: syntax error, unexpected '=', expecting '('
